src/translators/beamoji_emojilist_translator.erl

```diff
@@ -0,0 +1,64 @@
+-module(beamoji_emojilist_translator).
+
+-behaviour(beamoji_translator).
+
+-ifdef(TEST).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-endif.
+
+-export(['🐣'/0, '⏩'/2, '⏪'/2]).
+
+-record(state, {'⏩' :: #{binary() => binary()}, '⏪' :: #{binary() => binary()}}).
+
+-spec '🐣'() -> beamoji_translator:'🗺'().
+'🐣'() ->
+    Path = filename:join([code:priv_dir(beamoji), "emoji.eterm"]),
+    {ok, [#{} = ShortToEmoji]} = file:consult(Path),
+    Emojis = maps:values(ShortToEmoji),
+    Shorts = maps:keys(ShortToEmoji),
+    EmojiToShort =
+        maps:from_list(
+            lists:zip(Emojis, Shorts)),
+    #state{'⏩' = ShortToEmoji, '⏪' = EmojiToShort}.
```

Comment:
This should be a map to match with the type expected by `beamoji_translator`.

Reply:
done